libshaderc_spvc/src/spvc.cc

```diff
@@ -381,13 +380,13 @@ shaderc_spvc_status shaderc_spvc_unset_decoration(
 
 inline void shaderc_spvc_for_each_combined_image_sampler(
     const shaderc_spvc_context_t context,
-    void (*f)(uint32_t*, uint32_t*, uint32_t*)) {
+    void (*f)(uint32_t, uint32_t, uint32_t)) {
   for (const auto& combined :
        context->cross_compiler->get_combined_image_samplers()) {
     uint32_t x = combined.sampler_id;
```

Comment:
nit: There is no need have these temp variables. You can write f(combined.sampler_id, combined.image_id, combined.combined_id);

Reply:
Yes, that's much better (sorry about these nits, they're left over from from other approaches I tried).